### DISCUSSION TITLE
CSS Slector to change font of HTML Pages

### DISCUSSION THREAD

**Discussion by v3ss0n** [Jelmerro/Vieb#417] [General]
I am tryin to change font , i could change the UI font but cannot change the HTML/Website font.

Here is my attempt.

```
/* general */
#app {flex-direction: column-reverse;}
#url-hover {bottom: 4em;}
#suggest-dropdown {top: auto;bottom: 2.1em;flex-direction: column-reverse;}
#mode-suggestions {top: auto;bottom: 2em;flex-direction: column-reverse;}
.bottomright {right: .7em;bottom: 4em;;}
.bottomleft {left: .7em;bottom: 4em;}
.topright {right: .7em;top: .7em;}
.topleft {left: .7em;top: .7em;}
#app.tabshidden .topright, #app.tabshidden .topleft {top: .7em;}
#app.navigationhidden .topright, #app.navigationhidden .topleft {top: .7em;}
#app.tabshidden .bottomright, #app.tabshidden .bottomleft {bottom: 2em;}
#app.navigationhidden .bottomright, #app.navigationhidden .bottomleft {bottom: 2em;}
#app.navigationhidden.tabshidden .bottomright, #app.navigationhidden.tabshidden .bottomleft {bottom: 0;}
/* hidden bars */
#app.tabshidden #url-hover, #app.navigationhidden #url-hover {bottom: 2em;}
#app.tabshidden.navigationhidden #url-hover {bottom: 0;}
#app.navigationhidden #suggest-dropdown {top: auto;bottom: .1em;}

html body *{
    font-family: IBM Plex Sans;
    font-size: 18px;
}

```

How can i override the WebPage font?

**Comment by Jelmerro** [comment]
The Vieb colorschemes are for styling Vieb itself, not for the pages, see `:help colorscheme`. You can style the pages itself with the `userstyle` setting, see `:help userstyle`. Let me know if you need more help with either of them.

**Reply by v3ss0n** [reply]
Nice , checking it now!

**Reply by v3ss0n** [reply]
Thanks a lot , i am able to do it.
If i want to include a customfont , can i just put there and do`@fontface` with`local()` ? i am new to electron and extesion development so i am checking. Or if i need to upload to a cdn.-

**Reply by Jelmerro** [reply]
I haven't tried using custom fonts, but if you install it as a user or system font you probably don't need need any fontface or local magic, you can just refer to it by name and it should work.

**Reply by v3ss0n** [reply]
yeah if preinstalled , it works , i am thinking about distributing user styles plus font so other can use if installed or not.
